src/executors/python-default.yml

```
@@ -0,0 +1,23 @@
+description: Common cimg/node image executor
+parameters:
+  repo:
+    type: string
+  tag:
+    default: "3.7"
+    type: string
+  resource_class:
+    default: large
+    description: Configure the executor resource class
+    enum:
+      - small
+      - medium
+      - medium+
+      - large
+      - xlarge
+      - 2xlarge
+      - 2xlarge+
+    type: enum
+working_directory: ~/repo
+docker:
+  - image:  << parameters.repo >>/python:<< parameters.tag >>
```

If you make the change above then we should update this too
```suggestion
  - image:  << parameters.repo >>:<< parameters.tag >>
```